application/actions/actions.go

```diff
@@ -122,10 +129,20 @@ func getExtras(c buffalo.Context) map[string]interface{} {
 func getUUIDFromParam(c buffalo.Context, param string) (uuid.UUID, error) {
 	s := c.Param(param)
 	id := uuid.FromStringOrNil(s)
+	newExtra(c, param, s)
+
 	if id == uuid.Nil {
-		newExtra(c, param, s)
 		err := fmt.Errorf("invalid %s provided: '%s'", param, s)
 		return uuid.UUID{}, api.NewAppError(err, api.ErrorMustBeAValidUUID, api.CategoryUser)
 	}
 	return id, nil
 }
+
+type handlerData struct {
+	User     models.User
+	ModelObj interface{}
```

Comment:
I was imagining that each handler group would have its own typo so they wouldn't need to use `interface{}`. But this works too.

Reply:
I think you're right.